--- a/examples/connection_tester.cpp
+++ b/examples/connection_tester.cpp
@@ -46,6 +46,14 @@
 #include <boost/array.hpp>
 #include <boost/detail/atomic_count.hpp>
 
+#if BOOST_ASIO_DYN_LINK
+#if BOOST_VERSION >= 104500
+#include <boost/asio/impl/src.hpp>
+#elif BOOST_VERSION >= 104400
+#include <boost/asio/impl/src.cpp>
+#endif
+#endif
+
 using namespace libtorrent;
 using namespace libtorrent::detail; // for write_* and read_*
